src/libs/deploy.js

```diff
@@ -189,6 +189,16 @@ export const deployAppleStore = async (signedApp, workspace = '/tmp/') => {
  */
 // eslint-disable-next-line import/prefer-default-export
 export const deployAirWatch = async (signedApp, platform, awOrgID, awFileName, workspace = '/tmp/') => {
+  // The urls for airwatch api:
+  const awHost = process.env.AIRWATCH_HOST;
```

Comment:
We're using config to access environment and other variables. Lets move these into an `airwatch: {}` section and use `config.get('airwatch:host)` (for example).